ci/run_cudf_polars_polars_tests.sh

```diff
@@ -48,7 +48,9 @@ python -m pytest \
        --cache-clear \
        -m "" \
        -p cudf_polars.testing.plugin \
-       -v \
+       -n 8 \
+       --dist=worksteal \
+       -vv \
```

Comment:
Did you necessarily want to increase the verbosity for these tests?

Reply:
Yes, to be able to see if there is a slowdown/hangs in tests. This has been very helpful in pandas pytests job.